A <%= exception.class.to_s %> occured: <%= exception.message.to_s %>
<% if exception.respond_to?(:backtrace) && exception.backtrace.present? %>
File:                    <%=   exception.backtrace.slice(0, 1).join("\n  ")  %>
<% end %>
PID:                     <%= process %>
PWD:                     <%= pwd %>
Uname:                   <%= uname %>
Server:                  <%= server %>
Error time:	             <%= timestamp %>
<% if request_path.present? %>
Request URI:             <%= request_path %>
<% end %>
<% if referrer.present? %>
Referer:             <%= referrer %>
<% end %>
<% if user_agent.present? %>
User Agent:	             <%= user_agent %>
<% end %>
